runner.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { spawn } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+const merge = require('./merge_reports')
+const Bandit = require('./linters/bandit')
+const Gosec = require('./linters/gosec')
+
+/**
+ * Run all linters on specified files
+ * @param {string[]} files Files to analyze
+ * @param {string} repoID
+ * @param {string} prID
+ */
+async function runLinters (files, repoID, prID) {
+  // TODO: Sync with file download location resolution
```

Comment:
Also in the future, it would be nicer if the linters were pluggable and not hardcoded here.  I imagine a linter registration model, that the runner would iterate through the loaded linters.

Reply:
I thought the same yes. That would be really cool.

Reply:
Yes, this is a rough first draft but it would be nice to have an array of objects and just iterate through it. I'll see what I can do.

Reply:
`linters/index.js` now exports a list of code linters, we might add a utility function to register linters dynamically later

Reply:
Note that this requires the report generation rewrite, coming in a followup PR